index.html

```diff
@@ -46,6 +46,150 @@ <h3>LET'S CONNECT</h3>
                     </ul>
                 </div>
             </section>
+            
+            <!-- Works Section -->
+            <section class="works-section">
+                <div class="project">
+                    <img src="/images/snapshoot-portfolio-1.svg" alt="">
+                    <h3>Tonic</h3>
+                    <div class="canopy">
+                        <p class="title">CANOPY</p>
+                        <img class="counter" src="images/Counter.png" alt="counter" />
+                        <p class="job">Back End Dev</p>
+                        <img class="counter" src="images/Counter.png" alt="counter">
+                        <p class="date">2015</p>
+                    </div>
+                    <p class="project-description">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <ul class="project-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
```

Comment:
- [ ] Works section is looking good, great job! I would suggest adding more space between the description and the languages:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-04-29 a la(s) 10 14 22 p  m](https://user-images.githubusercontent.com/83717285/166088813-ce81d1b7-3ebb-4ea7-a277-06f65e783aa3.png)|![Captura de pantalla 2022-04-29 a la(s) 10 14 13 p  m](https://user-images.githubusercontent.com/83717285/166088826-08256f77-2ad7-436b-a116-7b8a72427002.png)|